Makefile

```diff
@@ -1,4 +1,4 @@
-PYTHON_VERSION_VENV ?= python3.9
+PYTHON_VERSION_VENV ?= python3
```

Comment:
Personally I will keep using py3.9 for development until we drop support for it (but I'm fine with that not being the default, I can set it locally)

(I prefer earlier feedback that what I'm using doesn't exist in 3.9 as opposed to finding out only once I run the unit tests)

Reply:
And I found out that the issue in f1915e55e7ee115a5aed0ebd8581b070bb608f5b was in fact caused by tox misconfiguration. The way we have it configured, we don't run tests for py3.9, py3.10 and py3.11, we run tests for [whatever version of python you use to invoke `tox`] 3 times

I can fix that in a separate PR

Reply:
go ahead if you want, but I have those changes on a separate branch as the changes were not  strictly related to the Makefile

Reply:
ah great, I'll leave it to you then :slightly_smiling_face: